testing/spectest/shared/bellatrix/operations/deposit.go

```diff
@@ -23,5 +23,5 @@ func blockWithDeposit(ssz []byte) (interfaces.SignedBeaconBlock, error) {
 }
 
 func RunDepositTest(t *testing.T, config string) {
```

Comment:
I have a doubt if we are making it common for all forks shouldn't we keep the tests for it inside blocks pkg only as it doesn't depends on version ?